platform/commonUI/edit/src/actions/SaveAsAction.js

```diff
@@ -160,7 +160,7 @@ function (
         }
 
         function saveAfterClone(clonedObject) {
-            return this.openmct.editor.save().then(() => {
+            return self.openmct.editor.save().then(() => {
```

Comment:
Yeeesh, yep, this would have come up as a bug when we eventually package Open MCT as an ES6 module.